Boa.Constrictor.UnitTests/Screenplay/Caching/AnswerCacheTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Boa.Constrictor.Screenplay;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+
+namespace Boa.Constrictor.UnitTests.Screenplay
+{
+    [TestFixture]
+    public class AnswerCacheTest
+    {
+        #region Variables
+
+        private IActor Actor { get; set; }
+        private AnswerCache Cache { get; set; }
+        private Mock<IQuestion<int>> MockQuestionA { get; set; }
+        private Mock<IQuestion<int>> MockQuestionB { get; set; }
+
+        #endregion
+
+        #region Setup
+
+        [SetUp]
+        public void SetUp()
+        {
+            Actor = new Actor();
+            Cache = new AnswerCache();
+
+            MockQuestionA = new Mock<IQuestion<int>>();
+            MockQuestionA.Setup(x => x.RequestAs(It.IsAny<IActor>())).Returns(1);
+            MockQuestionA.Setup(x => x.GetHashCode()).Returns(1);
+
+            MockQuestionB = new Mock<IQuestion<int>>();
+            MockQuestionB.Setup(x => x.RequestAs(It.IsAny<IActor>())).Returns(2);
+            MockQuestionB.Setup(x => x.GetHashCode()).Returns(2);
+        }
+        
+        #endregion
+
+        #region Tests
+
+        [Test]
+        public void GetCached()
+        {
+            Cache.Get(MockQuestionA.Object, Actor);
+            Cache.Get(MockQuestionB.Object, Actor);
```

Comment:
Did you want to validate B's answer in this test?

Reply:
Not necessarily. I just wanted the cache to have more than one entry.